src/Discovery/DiscoveryActionBar.tsx

```diff
@@ -483,7 +502,9 @@ const DiscoveryActionBar = (props: Props) => {
           arrowPointAtCenter
           content={(
             <React.Fragment>
-          Directly download data (up to 250Mb) from selected studies
+              {healICPSRLoginNeeded
+                ? 'This dataset is only accessible to users who have authenticated via InCommons. Please log back in using the InCommons option.'
```

Comment:
```suggestion
                ? 'This dataset is only accessible to users who have authenticated via InCommons. Please log in using the InCommons option.'
```
Changed my mind with this minor text update